.github/workflows/canary-py.yml

```diff
@@ -0,0 +1,85 @@
+name: canary-python
+env:
+  STACK: py38-canary
+  UTILS: ${{ github.workspace }}/opentelemetry-lambda/utils/sam
+
+on:
+  schedule:
+    - cron: '15 * * * *' # every hour
+  workflow_dispatch:
+
+jobs:
+  canary-test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        region: [ us-east-1, us-east-2, us-west-1, us-west-2, ap-south-1, ap-northeast-3, ap-northeast-2, ap-southeast-1, ap-southeast-2, ap-northeast-1, ca-central-1, eu-central-1, eu-west-1, eu-west-2, eu-west-3, eu-north-1, eu-south-1]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Checkout submodules
+        run: |
+          git submodule update --init --recursive
+          cd opentelemetry-lambda
+          git fetch
+          git merge origin/master
```

Comment:
We want to manage the submodules version in the repository, at least for normal workflows (we could have an upstream-head type of build if we want it). So we shouldn't need these steps, just `submodule: true` in the checkout action.

Reply:
I see, thanks.